plugins/Previous/README.md

```diff
@@ -0,0 +1,7 @@
+# @hookstate/devtools
+
+[![license](https://img.shields.io/github/license/avkonst/hookstate)](https://img.shields.io/github/license/avkonst/hookstate) [![npm version](https://img.shields.io/npm/v/@hookstate/previous.svg?maxAge=300&label=version&colorB=007ec6)](https://www.npmjs.com/package/@hookstate/devtools)
+
+Previous state for @hookstate/core.
+
+[Learn more...](https://hookstate.js.org/docs/previous)
```

Comment:
I suppose it will be be `https://hookstate.js.org/docs/extensions-previous` for consistency with other plugins.